packages/service/src/service.ts

```diff
@@ -50,9 +59,7 @@ app.post("/document", function (req, resp) {
   resp.send({ user: req.user });
 });
 app.get("/documents", async function (req, resp) {
-  if (!req.user) {
-    resp.status(401);
-    resp.send("need to login");
+  if (!handleAuthCheck(req, resp)) {
```

Comment:
I wonder if we can simplify to something like this:
```ts
function auth(req, resp, next){
  if (!req.user) {
    resp.status(401);
    next(new Error('need to login'))
  } else {
    next();
   }
}

app.get("/documents", 
  auth,
  async function (req, resp) {
    ...
  }
)
```

Check the doc for these express routes https://expressjs.com/en/4x/api.html#router